src/nwnsdk/postgres/postgres_client.py

```diff
@@ -17,41 +17,38 @@ def __init__(self, postgres_config: PostgresConfig):
         initialize_db("nwn", postgres_config)
 
     def send_input(
-        self, job_id: uuid4, job_name: str, work_flow_type: WorkFlowType, esdl_str: str, user_name: str
+            self, job_id: uuid4, job_name: str, work_flow_type: WorkFlowType, esdl_str: str, user_name: str
     ) -> None:
         with session_scope() as session:
             new_job = Job(
                 job_id=job_id,
                 job_name=job_name,
-                work_flow_type=work_flow_type.value,
-                map_editor_user=user_name,
-                status="registered",
+                work_flow_type=work_flow_type,
+                user_name=user_name,
+                status=JobStatus.REGISTERED,
                 input_esdl=esdl_str,
                 added_at=datetime.now(),
             )
             session.add(new_job)
 
     def retrieve_input_esdl(self, job_id: uuid4) -> str:
-        session: Session
         LOGGER.debug("Retrieving esdl for job %s", job_id)
         with session_scope() as session:
-            stmnt = select(Job.input_esdl).where(Job.job_id == job_id)
-            input_esdl: str = session.scalar(stmnt)
-        return input_esdl
+            stmnt = select(Job).where(Job.job_id.is_(job_id))
```

Comment:
Please revert to only retrieving the esdl. Currently the whole job is retrieved.